src/Angular.js

```diff
@@ -893,7 +893,35 @@ var csp = function() {
   return (csp.isActive_ = active);
 };
 
+/**
+ * @ngdoc directive
+ * @name ng.directive:ngJq
+ *
+ * @element html
+ * @param {string} (optional) the name of the library available under `window` to be used for angular.element
```

Comment:
@mzgol Look again, it's already there.  The inline comment seems to be referencing the old diff.  If you look at the file directly from the PR, you'll see that the optional flag is there.